src/tests/SearchResults.test.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
```

Comment:
- [ ] It will be great to fix the eslint errors. Kindly don't disable eslint in your files.